### DISCUSSION TITLE
Changes made to config.toml to have cargo build from source on M2

### DISCUSSION THREAD

**Discussion by rawkakani** [denoland/deno#19710] [General]
Hi I wanted to ask what I am doing wrong 

I had to change my  `./cargo/config.toml` file from

```
[target.aarch64-apple-darwin]
rustflags = ["-C", "link-arg=-fuse-ld=lld"]
```

 to 
```
[target.aarch64-apple-darwin]
rustflags = [
"-C", "link-arg=-undefined",
"-C", "link-arg=dynamic_lookup",
]
```

using a M2 chip and following the build from source documentation deno provides to be able to successfully build deno and run 

`./target/debug/deno run cli/tests/testdata/run/002_hello.ts` 

was wondering if anyone had any insight as to the pitfalls to the changes I made

**Comment by bartlomieju** [comment]
You shouldn't need to change this file if you have LLD installed, you can do it by running
```
brew install llvm
# Add /opt/homebrew/opt/llvm/bin/ to $PATH
```

There will probably be no big difference with your setting, but we use `lld` to speed up linking.

**Reply by rawkakani** [reply]
noted thank you
